common/lib/capa/capa/templates/drag_and_drop_input.html

```diff
@@ -17,14 +17,14 @@
     <input type="text" name="input_${id}" id="input_${id}" aria-describedby="answer_${id}" value="${value|h}"
     style="display:none;"/>
 
-    <p class="status" aria-describedby="input_${id}">
-    ${status.display_name}
+    <p class="status drag-and-drop--status" aria-describedby="input_${id}">
+        ${status.display_name}
     </p>
 
     <p id="answer_${id}" class="answer"></p>
 
     % if msg:
-        <span class="message">${HTML(msg)}</span>
+        <span class="message" aria-describedby="answer_input_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
"answer_input_${id}" does not exist anyplace else in this template, and thus is most likely not a legitimate value for aria-describedby.